source/_components/asterisk_mbox.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Asterisk Voicemail"
+description: "Instructions how to integrate your existing Asterisk voicemail within Home Assistant."
+date: 2017-06-30 18:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Other
+ha_iot_class: "Local Push"
+---
+
+The Asterisk Voicemail integration for Home Assistant allows you to view, listen-to, and delete voicemails from a Asterisk voicemail mailbox.  The component includes a panel on the frontend that provides caller-id and speech-to-text transcription (using Google's API) of messages in addition to playback and message deletion.  There is also an included sensor that provides an indication of the number of available messages.  There is no requirement that the Asterisk PBX and Home Assistant are running on the same machine.
+
+To enable the plugition is required in both Home Assistant as well as on the asterisk server.
+
+First follow the [Asterisk PBX configuration guide](/docs/asterisk_mbox) to setup the necessary server on the Asterisk PBX server (this is necessary even if Asterisk and Home Assistant are running on the same server)
+
+Once that is complete, add the the following entry `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+asterisk_mbox:
+    password: YOUR_PASSWORD
```

Comment:
Recommend changing the variable from `YOUR_PASSWORD` to `ASTERISK_PBX_PASSWORD`.